packages/core/src/browser/label-provider.ts

```diff
@@ -106,6 +143,20 @@ export namespace URIIconReference {
 @injectable()
 export class DefaultUriLabelProviderContribution implements LabelProviderContribution {
 
+    protected formatters: ResourceLabelFormatter[] = [];
+    protected readonly onDidChangeEmitter = new Emitter<DidChangeLabelEvent>();
+    protected homePath: string | undefined;
+    constructor(@inject(EnvVariablesServer) protected readonly envVariablesServer: EnvVariablesServer) {
```

Comment:
please use property injection and `postConstruct` annotation: https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#property-injection

Reply:
done